_includes/latest_posts.html

```diff
@@ -14,7 +14,18 @@
                 <tr>
                   <th scope="row">{{ item.date | date: "%b %-d, %Y" }}</th>
                   <td>
-                    <a class="news-title" href="{{ item.url | relative_url }}">{{ item.title }}</a>
+                    {% if item.redirect == blank %}
+                      <a class="news-title" href="{{ item.url | relative_url }}">{{ item.title }}</a>
+                    {% else %}
+                      {% if item.redirect contains '://' %}
```

Comment:
nit: could you switch to using `elsif` pattern here instead of nested if-then-else? here's an example: https://github.com/alshedivat/al-folio/blob/master/_includes/head.html#L27

Reply:
@alshedivat Done